babel.config.js

```diff
@@ -38,6 +38,19 @@ const noModuleTarget = {
     : ['Last 2 versions'],
 };
 
+const plugins = [
+  './build-system/babel-plugins/babel-plugin-transform-fix-leading-comments',
+  '@babel/plugin-transform-react-constant-elements',
+  [
+    '@babel/plugin-transform-react-jsx',
+    {
+      pragma: 'Preact.createElement',
+      pragmaFrag: 'Preact.Fragment',
+      useSpread: true,
+    },
+  ],
+];
+
```

Comment:
Nice. Could you also pull these out to `presets` and `esmPresets`?

https://github.com/ampproject/amphtml/blob/58a99467b92a6af72a6c47a2bf57fd5fdd2ad060/babel.config.js#L62-L78